src/horust/healthcheck.rs

```diff
@@ -244,11 +240,10 @@ file-path = "{}""#,
     }
 
     #[test]
-    fn test_http_healthiness_check() -> Result<()> {
+    fn test_http_healthiness_check() {
         thread::spawn(move || {
             start_server();
         });
         assert_eq!(check_http_endpoint("http://127.0.0.1:9999"), true);
```

Comment:
Still binding to a fixed port. Please use a random port instead.

Reply:
Yes. I am facing a problem combining a str and u16. Any idea how to do it?

Reply:
To concat string and stuff, just use format!("http://127.0.0.1:{}", port).
Please keep in mind that you should use a random **available** port. Check [this](https://rust-lang-nursery.github.io/rust-cookbook/net/server.html) as an example.

Reply:
I referred the link you shared for random port number generation I am looking into it. In latest commit i have used `Healthiness` and `healthchecks`.